testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HadoopKerberos.java

```diff
@@ -73,8 +73,8 @@ public void extendEnvironment(Environment.Builder builder)
         });
         builder.configureContainer(TESTS, container -> {
             container.setDockerImageName(dockerImageName);
-            container.withCopyFileToContainer(forHostPath(configDir.getPath("tempto-configuration.yaml")), CONTAINER_TEMPTO_PROFILE_CONFIG);
         });
+        configureTempto(builder, configDir, "hadoop-kerberos");
```

Comment:
this filedescription is nice, but it's not copy-paste-friendly.  would you consider dropping it?